File: Cpp Solutions/sol1.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    int T;
    cin >> T;

    while (T--) {
        int N;
        cin >> N;

        vector<int> L(N);

        for (int i = 0; i < N; ++i) {
            cin >> L[i];
        }

        int evenCount = 0;
        int oddCount = 0;

        for (int i = 0; i < N; ++i) {
            if (L[i] % 2 == 0) {
                evenCount++;
            } else {
                oddCount++;
            }
        }

        if (evenCount > oddCount) {
            cout << "Yes" << endl;
        } else {
            cout << "No" << endl;
        }
    }

    return 0;
}

File: Python Solutions/sol1.py
def check_lights_count(n, lights):
    even_count = 0
    odd_count = 0
    
    for light in lights:
        if light % 2 == 0:
            even_count += 1
        else:
            odd_count += 1
    
    if even_count > odd_count:
        return "Yes"
    else:
        return "No"

T = int(input())

for _ in range(T):
    N = int(input())  
    L = list(map(int, input().split()))  
    
    result = check_lights_count(N, L)
    print(result)
